src/com/oltpbenchmark/schema/TableSchema.java

```diff
@@ -0,0 +1,60 @@
+package com.oltpbenchmark.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TableSchema {
+    private String name;
```

Comment:
should be `final` and initialized via constructor.

Reply:
Done